backend/src/services/auth/auth.service.ts

```diff
@@ -15,8 +16,13 @@ class Auth {
     this.#userService = userService;
   }
 
-  signUp(userRequestDto: UserSignUpRequestDto): Promise<UserSignUpResponseDto> {
-    return this.#userService.create(userRequestDto);
+  async signUp(
+    userRequestDto: UserSignUpRequestDto,
+  ): Promise<UserSignUpTokenResponseDto> {
+    return {
+      token: createToken({ data: userRequestDto }),
```

Comment:
```suggestion
      token: this.toketService.create({ data: userRequestDto }),
```
create a token service, instead helpers